reemgawad/aircams
# Setup devise

- What has changed
Setup Devise
added firstName and Name to Users Model

- How to test 
see logging at http://localhost:3000/users/sign_up
check schema of User Model

@reemgawad
